packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -254,4 +319,94 @@ - (UIImage *)scaledImage:(UIImage *)image
   return scaledImage;
 }
 
+- (void)createImageFileAtPath:(NSString *)path contents:(NSData *)data {
+  if ([[NSFileManager defaultManager] createFileAtPath:path contents:data attributes:nil]) {
+    _result(path);
+  } else {
+    _result([FlutterError errorWithCode:@"create_error"
+                                message:@"Temporary file could not be created"
+                                details:nil]);
+  }
+  _result = nil;
+  _arguments = nil;
+}
+
+- (void)createImageFileAtPath:(NSString *)path contents:(NSData *)data with:(PHAsset *)asset {
+  NSMutableDictionary *exifDict = [self fetchExifFrom:asset];
```

Comment:
Manually parsing the meta data seems to be risky. Is it possible to get the meta data from the original image directly and pass down to the new image? 
Maybe when fetching image in `requestImageDataForAsset`, we can call `(NSDictionary *)CFBridgingRelease(CGImageSourceCopyPropertiesAtIndex(cgImage,0,NULL));` to get the whole meta data of the old image, and then somehow get a diff between the new image and the old image, then add the necessary stuff to the new image?
This way we don't manually parse the information like formatting time, adding hard coded GPS keys etc.